force-app/main/default/classes/UTIL_HtmlOutput_TEST.cls

```diff
@@ -122,37 +94,4 @@ public with sharing class UTIL_HtmlOutput_TEST {
         System.assert(actualHtml.startsWith('&lt;'));
         System.assertNotEquals(html, actualHtml);
     }
-
-    /*******************************************************************************************************
-    * @description Verifies that non-allowed javascript event code is stripped from url
-    */
-    @isTest
-    private static void shouldStripJavaScriptFromUrl() {
```

Comment:
@balsamhindi why are we removing this test? Any string could be used for a URL here and should not be related to this power of us hub link work since this is in a test class so it won't ever appear on any UI. This should be added back.

Reply:
After discussion with Daniel I've gone ahead and taken the route of changing the url to the corresponding trailhead one. Want to make sure Erin is aware of this as I know in our initial discussion that wasn't preferred but as a note this won't show up and reflect in the UI at all, it's just needed to make the tests run properly.

Reply:
That's perfectly fine, @balsamhindi. We're trying to limit the URLs to avoid work for the devs in the future, so if this is a good dev solution, I'm 100% okay with it. Thanks for checking!

Reply:
Thank you for verifying that Erin, yeah I know you mentioned you've already had to change the URLs so often, but at the very least this is a test file so hoping that allows for minimal changes in that context!